test/app/components/content_for_component.html.erb

```diff
@@ -1,2 +1,6 @@
-<% content_for(:foo, "bar") %>
-<div>Hello content for</div>
+<%# content_for(:foo, "bar") %>
```

Comment:
as written uncommenting this content_for breaks because `:foo` isn't an attribute of the component. As mentioned in #144 I think we likely don't want to have a naming collision to avoid confusion like this.

Reply:
Let's be sure to uncomment this before moving forward. This should continue to work.

Reply:
done